bin/asd-prereq-checker.sh

```diff
@@ -80,6 +76,14 @@ function check_org_policy_is_valid {
     exit 1
   fi
 
+  result=$(gcloud beta resource-manager org-policies describe compute.trustedImageProjects --project=$PROJECT_ID --effective)
+  if [[ "$result" == *"DENY"* ]]; then
+    echo
+    echo $INVALID_ORG_POLICY_TRUSTED_IMAGES
+    echo
+    exit 1
+  fi
+
   echo "PASS: Org Policy will allow this deployment."
 }
 
```

Comment:
This function is generating additional outputs and exiting the prereq script as soon as one org policy prereq does not pass. Maybe something like this? Just an example
```
function check_org_policy_is_valid {
  # to suppress output  
  gcloud beta resource-manager org-policies list --project=$PROJECT_ID >/dev/null 2>&1 
  if [[ $? >0 ]]; then
    echo "WARNING: Unable to verify if the project has any Organization Policies that will prevent the deployment."
    return
  fi

  # to keep track of any failures
  fail=0

  result=$(gcloud beta resource-manager org-policies describe compute.requireOsLogin --project=$PROJECT_ID --effective)
  if [[ "$result" == *"enforced: true"* ]]; then
    echo
    echo $INVALID_ORG_POLICY_OSLOGIN
    echo
    fail=1
  fi

# some blocks omitted here

  if [[ $fail == 1 ]]; then
    return
  fi
  echo "PASS: Org Policy will allow this deployment."
}
```

Reply:
Updated to the Org Policy permission check to not output anything. We could make those changes to check all Org Policy at once, but I don't think it's required. The other checks will all exit the script as soon as something fails.

Reply:
right, looks like check_iam_policy is the only one that will proceed with warning now, which makes sense.